src/components/RecoverPass/index.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+//Import Material-UI
+import { Box, TextField, Button, Card, CardMedia, CardContent, Typography } from "@material-ui/core";
+//Import Formik
+import { useFormik } from "formik";
+//Import Styles
+import "./style.scss";
+//Import DATA
+import Logo from "assets/images/sushiMochi.jpg";
+
+//Import SchemaValidation
+import Schema from "./schema";
+
+const RecoverPass = () => {
+  const formik = useFormik({
+    initialValues: {
+      email: "",
+    },
+    validationSchema: Schema,
+    validateOnChange: false,
+    onSubmit: values => {
```

Comment:
A menos que sea estrictamente necesario, nunca se accede a elementos del DOM directamente en React. Para resetear el form com Formik se hace lo siguiente:

```suggestion
    onSubmit: (values, actions) => {
      actions.resetForm()
    }
    // O con destructuring
    onSubmit: (values, {resetForm}) => {
      resetForm()
    }
```

Reply:
Done!

Perfecto. Bueno saber que tenía esa accion.